demos/jans-tarp/src/options/oidcClients.tsx

```diff
@@ -157,9 +172,9 @@ export default function OIDCClients(data) {
                             </TableRow>
                         </TableHead>
                         <TableBody>
-                            {(data.data === undefined || data.data.length == 0) ?
+                            {(data === undefined || data.length == 0) ?
                                 <TableCell colSpan={6}><Alert severity="warning">No Records to show.</Alert></TableCell> :
-                                data.data.map((row) => (<Row key={row.clientId} row={row} />))
+                                data.map((row) => (<Row key={row.clientId} row={row} notifyOnDataChange={handleNotifyOnDataChange} />))
```

Comment:
Please check data length first. For example - data?.length ? data.map... : <>Not row found</>

Reply:
Please let it be as it is , it is for displaying user-info on console.

Reply:
done